subprojects/core/src/main/java/org/gradle/api/tasks/WriteProperties.java

```diff
@@ -107,22 +125,23 @@ public void setProperties(Map<String, Object> properties) {
      * </p>
      *
      * @param name Name of the property
-     * @param value Value of the property
+     * @param value Value of the property or provider for value
      * @since 3.4
      */
     public void property(final String name, final Object value) {
```

Comment:
🤔 We should probably deprecate this one. I wonder how we should mark such methods

Reply:
Decided that it's too popular to do so.

Reply:
I would still deprecate that, just leave it in code for now

Reply:
Decided to move it out of scope of the current PR:
* https://github.com/gradle/gradle/issues/29828